src/archiefvernietigingscomponent/fixtures/default_emails.json

```diff
@@ -32,5 +32,15 @@
       "body": "Geachte gemeentearchivaris {{ user }},\nIngevolge het bepaalde in artikel 8 van het Archiefbesluit 1995, zoals gemandateerd aan de hoofden van de organisatieonderdelen in de mandaatregeling van Gemeente {{ municipality }}, verklaar ik hierbij dat de vernietiging zoals omschreven in {{ list }} is uitgevoerd. De zaken zijn onherstelbaar vernietigd. U kunt de vernietigingslijst in PDF formaat via deze link downloaden: {{ link_report }}",
       "subject": "Zaken op vernietigingslijst vernietigd"
     }
+  },
+  {
+    "model": "emails.automaticemail",
+    "pk": 4,
+    "fields": {
+      "type": "update_required",
+      "body": "Beste, \n Uw vernietigingslijst {{destruction_list}} heeft uw attentie nodig!",
```

Comment:
Please use the body that was mentioned in the issue: https://github.com/maykinmedia/archiefvernietigingscomponent/issues/91#issuecomment-966070176

They didn't specify a subject, so we should probably ask it at the next meeting

Reply:
Yep, will include that - this was a placeholder for the moment anyways.